backend/main.py

```
@@ -0,0 +1,550 @@
+                        processing_time = time.time() - start_time
+                        return QueryResponse(
+                            answer=f"Sorry, I couldn't find any illustrations matching '{search_term}'. You can ask to see all of my art.",
+                            processing_time=processing_time,
+                            llm_used="image_search"
+                        )
+
+        # Default to AI-powered text response
+        if not retriever:
+            raise HTTPException(
+                status_code=503,
+                detail=f"AI service temporarily unavailable - app not properly initialized"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)